src/main/java/com/notably/logic/suggestion/handler/NewSuggestionArgHandler.java

```diff
@@ -40,29 +44,36 @@ public NewSuggestionArgHandler(Model model) {
      * @return Optional.empty()
      */
     public Optional<SuggestionGenerator> handleArg(String userInput) {
+        logger.info("Starting handleArg method inside NewSuggestionArgHandler");
         ArgumentMultimap argMultimap =
                 ArgumentTokenizer.tokenize(userInput, PREFIX_TITLE, PREFIX_JUMP);
 
         if (userInput.isBlank()) {
+            logger.warning("user input is blank");
             model.setResponseText(RESPONSE_MESSAGE);
             return Optional.empty();
         }
 
         if (!ParserUtil.arePrefixesPresent(argMultimap, PREFIX_TITLE)
                 || !argMultimap.getPreamble().isEmpty()) {
+            logger.warning(String.format(ERROR_MESSAGE_INVALID_COMMAND, model.getInput()));
             model.setResponseText(String.format(ERROR_MESSAGE_INVALID_COMMAND, model.getInput()));
             return Optional.empty();
         }
 
         String title = argMultimap.getValue(PREFIX_TITLE).get();
 
         if (!Title.isValidTitle(title) && !title.isBlank()) {
+            logger.warning(String.format(ERROR_MESSAGE_INVALID_TITLE, title));
             model.setResponseText(String.format(ERROR_MESSAGE_INVALID_TITLE, title));
         } else if (title.isBlank()) {
+            logger.warning("title is blank");
```

Comment:
This shouldn't be a warning? Since it's a suggestion and it's not really an invalid thing to have a blank title